templates/dashboard/support.html

```diff
@@ -0,0 +1,97 @@
+{% extends 'default.html' %}
+
+{% set active_page = 'dashboard' %}
+
+{% block title %}
+    Support
+{% endblock %}
+
+{% block head %}
+    <style>
+        .card-title {
+            font-size: 22px;
+            font-weight: 600;
+            margin-bottom: 3px;
+        }
+    </style>
+{% endblock %}
+
+{% block script %}
+    <script>
+        new Vue({
+            el: '#alias-group',
+            data: {
+                ticket_email: '{{ ticket_email }}'
+            },
+            methods: {
+                generateRandomAlias: async function(event){
+                    let result = await fetch('/api/alias/random/new', { method: 'POST'})
+                    if(result.ok){
+                        let data = await result.json();
+                        this.ticket_email = data.alias;
+                    }
+                }
+            }
+        });
+
+        $('.custom-file input').change(function (e) {
+            let files = [];
+            for (let i = 0; i < $(this)[0].files.length; i++) {
+                files.push($(this)[0].files[i].name);
+            }
+            $(this).next('.custom-file-label').html(files.join(', '));
+        });
+    </script>
+{% endblock %}
+
+{% block default_content %}
+
+    <div class="col pb-3">
+        <div class="card">
+            <div class="card-body">
+                <div class="card-title mb-3">Report a problem</div>
+                <div class="card-subtitle mt-2 mb-2 pl-3 border-left border-warning" style="border-left-width: 5px !important;" >
```

Comment:
Nits: I'd recommend to use `alert alert-info` class here instead of the `card-subtitle mt-2 mb-2 pl-3 border-left border-warning` and we can also remove the `style`